.github/workflows/build.yml

```diff
@@ -42,3 +66,29 @@ jobs:
     - name: Test if expected files have been installed
       working-directory: build/check_install_root
       run: diff -u ../../test/jni_list.txt <(du -a | tac | awk -F ' ' '{print $2}' | sort)
+
+  build-on-windows-for-cppcli:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: install djinni
+      run: Invoke-WebRequest -Uri https://github.com/cross-language-cpp/djinni-generator/releases/download/v0.5.0/djinni.bat -OutFile djinni.bat
+    - name: Configure cmake
+      run: cmake -S . -B build -DDJINNI_WITH_CPPCLI=ON -DDJINNI_STATIC_LIB=ON -DCMAKE_INSTALL_PREFIX=build/check_install_root -DDJINNI_EXECUTABLE="$(((Get-Location).Path) -replace "\\","/")/djinni.bat" -G "Visual Studio 16 2019"
```

Comment:
Because on Windows asdf is not available the installation of djinni and usage in cmake is kind of nasty.
Any ideas how to solve this are welcome!

Reply:
I was too lazy to add this yesterday... Thanks for pushing me! :P